<launch>

    <!-- Loading twist_mux.yaml file-->
    <node pkg="twist_mux" type="twist_mux" name="twist_mux">
        <rosparam command="load" file="$(find emergency_stop_recover_pa)/config/twist_mux.yaml" />
        <remap from="cmd_vel_out" to="/cmd_vel"/>
    </node>

    <!--calling emergency-recover MODULE -->
    <node pkg="emergency_stop_recover_pa" type="emergency_stop_recover" name="emergency_stop_recover" output="screen" respawn="true">
        <rosparam param="safetyregion">
            [0.15, 0.15,
             0.00,-0.15,
             0.15, 0.00,
            -0.15,-0.15,
             0.00, 0.15,
            -0.15, 0.00]
        </rosparam>
    </node>
</launch>
